playwright/tests/customers.spec.ts

```diff
@@ -232,7 +232,7 @@ test("TC: SALEOR_207 Issue a new gift card for the customer @e2e @customer", asy
   await giftCardsPage.issueGiftCardDialog.clickOkButton();
   await giftCardsPage.expectElementIsHidden(giftCardsPage.giftCardDialog);
   await giftCardsPage.expectSuccessBannerMessage("Successfully created gift card");
-  await giftCardsPage.expectElementIsHidden(giftCardsPage.successBanner);
+  await giftCardsPage.expectSuccessBanner();
```

Comment:
it's the duplication of `await giftCardsPage.expectSuccessBannerMessage("Successfully created gift card");` ?